### PR TITLE
[9.0][MIG] fetchmail_notify_error_to_sender

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Send notice on fetchmail errors
=========================

This module extends the functionality of fetchmail to allow you to automatically send a notification email to sender, when odoo can't correctly process the received email.

### CODE REVIEW

File: fetchmail_notify_error_to_sender/README.rst [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+===============================
 Send notice on fetchmail errors
 ===============================
 
-If fetchmail is not able to correctly route an email, the email is "silently" lost (you get an error message in server log).
-For example, if you configure odoo mail system to route received emails according to recipient address, it may happen users send emails to wrong email address.
-
-This module allows to automatically send a notification email to sender, when odoo can't correctly process the received email.
-
+This module extends the functionality of fetchmail to allow you to automatically send a notification email to sender, when odoo can't correctly process the received email.
```
Comment (eLBati) [inline fetchmail_notify_error_to_sender/README.rst]: Hi @mreficent why did you remove the previous 2 lines?

**Review by dreispt — APPROVED** [review]
LGTM

File: fetchmail_notify_error_to_sender/README.rst (hunk 1 above)
Comment (MiquelRForgeFlow) [reply fetchmail_notify_error_to_sender/README.rst]: I don't remember, it was long time ago. I put them again.

**Review by eLBati — APPROVED** [review]
Thanks
